index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Fork Me! FCC: Test Suite Template</title>
```

Comment:
It would be nice for your title to have meaningful text. You could try something that defines your page. You could try something like "Jack Ruby's Portfolio"

Reply:
Good job 👍 in reviewing and suggesting.